### PR TITLE
Added source ads.price

### PR DESCRIPTION
**Pull request by Yuri364**
| Q             | A
| ------------- | ---
| Bug fix?      |  No
| New feature?  | Yes

## Description

Добавление источника `ads.price` - Определение стоимости ТС

Issue #89 

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I wrote unit tests for my code
- [x] I have made changes in [CHANGELOG.md](https://github.com/avtocod/specs/blob/master/CHANGELOG.md) file

### CODE REVIEW

File: reports/default/json-schema.json [hunk 1]
```diff
@@ -58,6 +58,17 @@
...
         },
+        "currency_type": {
+            "type": "string",
+            "pattern": "[A-Z]+",
+            "minLength": 3,
+            "maxLength": 3,
+            "enum": [
+                "RUB",
+                "EUR",
+                "USD"
```
Comment (Reallife) [inline reports/default/json-schema.json]: `null`?

File: reports/default/json-schema.json [hunk 2]
```diff
@@ -58,6 +58,17 @@
             "minLength": 4,
             "maxLength": 128
         },
+        "currency_type": {
+            "type": "string",
+            "pattern": "[A-Z]+",
```
Comment (tarampampam) [inline reports/default/json-schema.json]: `pattern` is required when `enum` is used?

Comment (tarampampam) [reply reports/default/json-schema.json]: `minLength` and `maxLength` also

File: reports/default/json-schema.json [hunk 3]
```diff
@@ -6894,6 +6903,96 @@
...
+                                                    "integer",
+                                                    "null"
+                                                ],
+                                                "examples": [
+                                                    630000
+                                                ],
+                                                "fillable_by": [
+                                                    "ads.price"
+                                                ]
+                                            },
```
Comment (tarampampam) [inline reports/default/json-schema.json]: this value can be negative? if not - we should set minimal value to `0`

Comment (tarampampam) [reply reports/default/json-schema.json]: This recommendation can be applied to the all `integer` properties

**Comment by tarampampam** [comment]
@Yuri364 awaiting for your changes 👍

File: reports/default/json-schema.json (hunk 1 above)
Comment (tarampampam) [reply reports/default/json-schema.json]: nullable defenition? I think `null` should not be defined here

File: reports/default/json-schema.json (hunk 2 above)
Comment (Yuri364) [reply reports/default/json-schema.json]: Fix

File: reports/default/json-schema.json (hunk 3 above)
Comment (Yuri364) [reply reports/default/json-schema.json]: Fix

**Pull request merged**